src/Altinn.Platform/Altinn.Platform.Storage/Storage.Interface/Models/InstanceEvent.cs

```diff
@@ -65,4 +67,21 @@ public override string ToString()
             return JsonConvert.SerializeObject(this);
         }
     }
+
+    /// <summary>
+    /// Represents a container object with a list of events related to instances.
+    /// </summary>
+    /// <remarks>
+    /// This should be used only when an API endpoint would otherwise return a list of instance events.
+    /// Not when the list is a property of a separate class.
+    /// </remarks>
+    [JsonObject(ItemNullValueHandling = NullValueHandling.Ignore)]
+    public class InstanceEvents
+    {
+        /// <summary>
+        /// The actual list of instance events.
+        /// </summary>
+        [JsonProperty(PropertyName = "data")]
+        public List<InstanceEvent> Data { get; set; }
```

Comment:
What is the reasoning behind naming the json property data rather than something related to events'?

Reply:
Will the result json be in this format? And why is that prefered over just returning the list without locking it in a property?

`"data": [{eventData}, 
{eventData},
{eventData}] `

Reply:
I read [somewhere ](https://github.com/OWASP/CheatSheetSeries/blob/master/cheatsheets/AJAX_Security_Cheat_Sheet.md#always-return-json-with-an-object-on-the-outside)that when returning a JSON an API should always start with an object at the root. I have not studied why, but I do think it is more future proof. It means we can give the list properties without breaking integrations.

At first I landed on naming all these `data` properties the same across all array entities, but in retrospect I don't mind making them list content specific.

Reply:
I can't rename the `Applications.Data` property to `Applications.Applications`, but I can change JSON serialization to write `{ "applications": [] }` just fine.

Now taking suggestions for naming the property.

Reply:
I would suggest naming the class Application(s)List and the property applications if that is allowed. Anyways.. Resolving the comment and approving as only minor changes are left.

Reply:
Done.
The Class is now called `ApplicationList` and the property Applications, which is serialized into `applications`. Similar for the other list types; `InstanceList`, `InstanceEventList` etc